macros/utils/table_operations/make_temp_relation.sql

```diff
@@ -1,11 +1,32 @@
+{% macro _make_temp_relation(base_relation, suffix=none) %}
```

Comment:
I added _ prefix because I didn't want to override the original adapters' macros (otherwise I won't be able to call `dbt.make_temp_relation` line 6)

Reply:
Makes total sense
Can you perhaps call it `edr_make_temp_relation`?
Just because we used a similar convention elsewhere